### DISCUSSION TITLE
Failed init storage: Post "https://openapi.aliyundrive.com/adrive/v1.0/user/getDriveInfo": dial tcp: lookup openapi.aliyundrive.com on ************:53: no such host

### DISCUSSION THREAD

**Discussion by panghujiajia** [AlistGo/alist#4690] [General]
Failed init storage: Post "https://openapi.aliyundrive.com/adrive/v1.0/user/getDriveInfo": dial tcp: lookup openapi.aliyundrive.com on ************:53: no such host

**Comment by aspiration-github** [comment]
解决了吗？我现在也有这样的情况，ping openapi.aliyundrive.com 是可以ping通的

**Reply by panghujiajia** [reply]
好久没用alist了，现在infuse可以直接挂阿里云盘就没用了
